docs/single_model_mode.md

```diff
@@ -48,11 +54,90 @@ docker run -d --rm -v ${PWD}/models:/models -p 9000:9000 -p 9001:9001 openvino/m
 +--------------------------------+---------------------------------------------------------------------------------------------------------------------------------+
 | `--rest_port`                  | | the REST server port                                                                                                          |
 +--------------------------------+---------------------------------------------------------------------------------------------------------------------------------+
-@endsphinxdirective
 
+@endsphinxdirective
 
-### Notes
-- Publish the container's port to your host's **open ports**.
+- Publish the container's port to your host's **open ports**. 
 - In the command above, port 9000 is exposed for gRPC and port 9001 is exposed for REST API calls.
-- For preparing and saving models to serve with OpenVINO&trade; Model Server refer to the [Model Repository](models_repository.md) article.
 - Add model_name for the client gRPC/REST API calls.
+
+## Serving Multiple Models 
+
+To use a container with several models, you need an additional JSON configuration file defining each model. In the file, provide a 
+`model_config_list` array that includes a collection of config objects for each served model. The `name` and the `base_path` values of the model are required for every config object.
+
+
+```json
+{
+   "model_config_list":[
+      {
+         "config":{
+            "name":"model_name1",
+            "base_path":"/opt/ml/models/model1",
+            "batch_size": "16"
+         }
+      },
+      {
+         "config":{
+            "name":"model_name2",
+            "base_path":"/opt/ml/models/model2",
+            "batch_size": "auto",
+            "model_version_policy": {"all": {}}
+         }
+      },
+      {
+         "config":{
+            "name":"model_name3",
+            "base_path":"gs://bucket/models/model3",
+            "model_version_policy": {"specific": { "versions":[1, 3] }},
+            "shape": "auto"
+         }
+      },
+      {
+         "config":{
+             "name":"model_name4",
+             "base_path":"s3://bucket/models/model4",
+             "shape": {
+                "input1": "(1,3,200,200)",
+                "input2": "(1,3,50,50)"
+             },
+             "plugin_config": {"CPU_THROUGHPUT_STREAMS": "CPU_THROUGHPUT_AUTO"}
+         }
+      },
+      {
+         "config":{
+             "name":"model_name5",
+             "base_path":"s3://bucket/models/model5",
+             "shape": "auto",
+             "nireq": 32,
+             "target_device": "HDDL"
+         }
+      }
+   ]
+}
+```
+
+When the Docker container has the config file mounted, it can be started - the command is minimalistic, as arguments are read from the config file. 
```

Comment:
```suggestion
Once the Docker container has the path to your config file mounted, it can be started. This simplifies the `docker run` command, as arguments are now read from the config file. 
```